### PR TITLE
Modified parser.py to process the aborted failures

### PR DESCRIPTION
**Pull request by smitthakkar96**
Resolves issue: #31

### CODE REVIEW

**Comment by smitthakkar96** [comment]
Ohh the change was blind sorry about that

**Comment by smitthakkar96** [comment]
Ohh the change was blind sorry about that

**Comment by smitthakkar96** [comment]
Ignore the commit 9b70503b6625524c2ec026c51a4c1753e2d1c639

**Comment by nigelbabu** [comment]
You removed the change you made the first time. You still need that.

**Comment by smitthakkar96** [comment]
@nigelbabu why would I need it? The condition is **not in**.

**Comment by nigelbabu** [comment]
Ah, good point. I'd misread that.

File: fstat/parser.py [hunk 1]
```diff
@@ -6,31 +6,44 @@
...
+        db.session.add(failure_instance)
+        db.session.commit()
+    except IntegrityError:
+        db.session.rollback()
+
+
 def process_failure(url, job_name, build_info):
     text = requests.get(url, verify=False).text
     accum = []
+    if text.find("Finished: ABORTED") != -1:
```
Comment (smitthakkar96) [inline fstat/parser.py]: Can I have example of such job that was aborted manually or because new patch set was uploaded.

Comment (nigelbabu) [inline fstat/parser.py]: This one is manually aborted: https://build.gluster.org/job/centos6-regression/4901/console

Comment (smitthakkar96) [inline fstat/parser.py]: Also if possible can you give me the example of job that was aborted due to new patchset was uploaded.

Comment (smitthakkar96) [inline fstat/parser.py]: how about this 
``` python 
if text.find("Finished: ABORTED") != -1 and text.find("Aborted by") == -1:
```

**Comment by nigelbabu** [comment]
I'm testing this and I'm finding a few issues inconsistent with the way we want things to work.

In a job that's aborted we don't care if there was a test failure. We only care about which test caused the hang. I would go so far as to say do not log failures for aborted jobs. Just the last running test.

I found that [this job](https://build.gluster.org/job/regression-test-with-multiplex/44/consoleText) causes two failures to be indexed -- one for the aborted job (as expected) and one for the failure that's also part of this job.

**Comment by nigelbabu** [comment]
I'm still not a fan of how we're displaying the information. A test that causes a job to be aborted need not be one that fails outright. There are tests that only cause timeouts and not failures. The UI you've built implicitly assumes that the same jobs are likely to fail or cause timeouts. That's not the case. If you split them up into two instead, and mark them differently, you make the critical ones float to the top.

Potential UI

File: fstat/templates/failure_instance.html [hunk 2]
```diff
@@ -14,7 +14,7 @@
     <div class="jumbotron">
       <h1>Failure Instance Summary</h1>
     </div>
-    <h2>Failure Signature: <a href="http://git.gluster.org/cgit/glusterfs.git/tree/{{ failure.signature }}" target="_blank"><code>{{ failure.signature }}</code></a></h2>
+    <h2>Failure Signature: <a href="http://git.gluster.org/cgit/glusterfs.git/tree/{{ failure.signature }}" target="_blank"><code class="failure-signature-main">{{ failure.signature }}</code></a></h2>
```
Comment (nigelbabu) [inline fstat/templates/failure_instance.html]: Can you add the icon here as well?

**Pull request merged**